src/hotspot/share/runtime/globals.hpp

```diff
@@ -1220,16 +1220,9 @@ const int ObjectAlignmentInBytes = 8;
   product(bool, UseCompiler, true,                                          \
           "Use Just-In-Time compilation")                                   \
                                                                             \
-  product(bool, UseCounterDecay, true,                                      \
-          "Adjust recompilation counters")                                  \
-                                                                            \
   develop(intx, CounterHalfLifeTime,    30,                                 \
           "Half-life time of invocation counters (in seconds)")             \
                                                                             \
```

Comment:
Thanks for making these changes. However, it seems like the obsolete `CounterHalfLifeTime` was missed in this cleanup?

Reply:
@ysramakrishna Thanks and yes, it should also be removed. I'll make a new issue and PR.

Reply:
For reference: https://github.com/openjdk/jdk/pull/21034